source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Down/121_DML_PIMS_ACQ_CHKLST_ITEM_TYPE_Alter_Down.sql

```diff
@@ -0,0 +1,229 @@
+/* -----------------------------------------------------------------------------
+Alter the data in the PIMS_ACQ_CHKLST_ITEM_TYPE table.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2023-Mar-22  Initial version
+----------------------------------------------------------------------------- */
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Delete rows from the table
+DELETE 
+FROM   PIMS_ACQ_CHKLST_ITEM_TYPE
+WHERE  ACQ_CHKLST_ITEM_TYPE_CODE IN (N'S3CHEQCOPY', N'S3APPRCOPY', N'ADVPAYMENT8', N'S6CHEQCOPY', N'S6APPRCOPY', N'EASEMENTCPY');
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter the "BCTFANOTIF" type
+DECLARE @CurrCd NVARCHAR(20)
+SET     @CurrCd = N'BCTFANOTIF'
+
+SELECT ACQ_CHKLST_ITEM_TYPE_CODE
+FROM   PIMS_ACQ_CHKLST_ITEM_TYPE
+WHERE  ACQ_CHKLST_ITEM_TYPE_CODE = @CurrCd;
+
+IF @@ROWCOUNT = 1
+  BEGIN
+  UPDATE PIMS_ACQ_CHKLST_ITEM_TYPE 
+  SET    DESCRIPTION                = N'If this a total take with a surplus, then BCTFA notified'
+       , CONCURRENCY_CONTROL_NUMBER = CONCURRENCY_CONTROL_NUMBER + 1
+  WHERE  ACQ_CHKLST_ITEM_TYPE_CODE = @CurrCd;
+  END
```

Comment:
@dfilteau 

![image](https://user-images.githubusercontent.com/24322224/227370478-abf670a4-c4b5-49c5-8920-66208dc633d9.png)

According to the mockup above, I would expect the following to be in the alter up scripts:

DESCRIPTION = 'BCTFA notified'
HINT = 'Applicable if this is a total take with a surplus'

That's is not what I'm seeing here - could you please update your scripts? Thanks